src/stories/HsitorySelect/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import {Button, ButtonProps, HistorySelectionDialogTitle, SelectionModal} from '../../components';
+
+export interface HistorySelectionProps extends ButtonProps{}
+
+export default function HistorySelection() {

```

Comment:
HsitorySelect - look, there is a typo in the directory name :)

Reply:
Btw, can you explain me please why do you create HistorySelection component here, should not it be in the react components folder?
I mean, I see that this component unites and gathers two different components in one (Button & SelectionModal), but feels like smth wrong with here (my memory whispers me about Atomic Design Methodology). I would suggest to split this task into the chunks, to add stories of the inner components first (Button already is added, but no for others).